src/qt/utilitydialog.cpp

```diff
@@ -58,7 +58,8 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->helpMessage->setVisible(false);
     } else {
         setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
+        QString header = "Usage: bitcoin-qt [command-line options] [URI]\n\n"
+                         "Optional [URI] is a bitcoin address in BIP21 URI format.\n";
```

Comment:
Square brackets mean that URI option is optional. Either use "[URI]" or "Optional URI" below, not "Optional [URI]".
```suggestion
                         "Optional URI is a bitcoin address in BIP21 URI format.\n";
```

Reply:
Done